pos_event_calendar/models/pos_session.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2025 Moka
+# @author Damien Horvat <damien@moka.cloud>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+class PosSession(models.Model):
+    _inherit = "pos.session"
+
+    def _pos_ui_models_to_load(self):
+        
+        models = super()._pos_ui_models_to_load()
+        models.extend(
+            [
+                "event.event",
+                "event.event.ticket"
+                "event.tag",
+            ]
+        )
+        return models
+
+    @api.model
+    def _load_pos_data_models(self, config_id):
+       """Load the data to the pos.config.models"""
+
+       data = super()._load_pos_data_models(config_id)
+       data += ["event.event", "event.event.ticket", "event.tag"]
```

Comment:
```suggestion
       data += ["event.tag"]
```

```suggestion
```

These 2 are already added in `super` by the core module: https://github.com/odoo/odoo/blob/9463bfeb58fb40176ecf1131bac6627a1627d02c/addons/pos_event/models/pos_session.py#L11